cccs-build/superset/requirements.txt

```diff
@@ -5,8 +5,8 @@ authlib==0.15.4
 python-json-logger==2.0.2
 
 # Additional drivers for Superset (or SQLAlchemy) to use
-trino==0.318.0
-mysql-connector-python==8.0.26
-elasticsearch-dbapi==0.2.4
+trino==0.328.0
```

Comment:
Just checking the number here.

Reply:
@cccs-tom requested that we use this version of the python client